src/main/java/neqsim/PVTsimulation/simulation/SaturationPressure.java

```diff
@@ -31,7 +31,12 @@ public SaturationPressure(SystemInterface tempSystem) {
    * @return a double
    */
   public double calcSaturationPressure() {
-    getThermoSystem().isImplementedCompositionDeriativesofFugacity(false);
+    boolean changed = false;;
```

Comment:
This ensures that multiphase check is always done for saturation pressure?

Reply:
yes. The multiphase flash is a more thorough flash (but slower). I have added an issue: #887